queries.sql

```diff
@@ -8,3 +8,28 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT *FROM animals WHERE neutered = 'True';
 SELECT *FROM animals WHERE name != 'Gabumon';
 SELECT *FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+
+UPDATE animals SET species = 'unspecified';
+
+ROLLBACK;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon%';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+COMMIT;
```

Comment:
- [ ] This is a new transaction and separate from the transaction above it, So please add `BEGIN` before this transaction.
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Verify%20that%20change%20was%20made%20and%20persists%20after%20commit.), Please verify that change was made and persists after commit.